Common/include/interface_interpolation/CSlidingMesh.hpp

```diff
@@ -64,7 +64,7 @@ class CSlidingMesh final : public CInterpolator {
    * \return Number of points included in the element.
    */
   static int Build_3D_surface_element(const unsigned long *map, const unsigned long *startIndex,
-                                      const unsigned long* nNeighbor, const su2double *coord,
+                                      const unsigned long* nNeighbor, su2activematrix const& coord,
```

Comment:
```suggestion
                                      const unsigned long* nNeighbor, const su2activematrix& coord,
```
I know you like this ordering but this is not how most of the code is written... And it will probably be changed if you run clang-format.